src/organizations/actions/thunks.ts

```diff
@@ -60,9 +64,23 @@ export const getOrganizations = () => async (
     }
 
     const {orgs} = resp.data
+    const numOrgs = orgs.length
+    const orgsWithCloudProvider: Organization[] = orgs
+
+    // api.getOrgs returns all organizations associated with the currently active account.
+    // Assuming cloud user, and assuming one org returned (no multi-org in cloud yet), retrieve
+    // organization's cloud provider from new org API.
+    if (CLOUD && numOrgs === 1 && Object.keys(orgs[0]).length > 0) {
```

Comment:
this code uses an inference - it's inferring that if `Object.keys(orgs[0]).length > 0` then `orgs[0].id` exists. I think it would be better to be more explicit and just check if `orgs[0].id` exists, e.g.

```ts
if (CLOUD && numOrgs === 1 && orgs[0].id) {
```